Client/src/CSS/NavBar.css

```diff
@@ -0,0 +1,28 @@
+.navbar {
```

Comment:
Since there are UI changes to your webpage, you could include screenshots in your PR description to show a before/after or after changes. This help would show the improvements/differences made by this PR.

Reply:
Noted!